legal-api/src/legal_api/resources/business/business.py

```diff
@@ -77,5 +78,14 @@ def post():
         if not isinstance(bootstrap, RegistrationBootstrap):
             return {'error': babel('Unable to create Incorporation Filing.')}, HTTPStatus.SERVICE_UNAVAILABLE
 
-        RegistrationBootstrapService.register_bootstrap(filing_account_id, bootstrap)
+        try:
+            business_name = json_input['filing']['incorporationApplication']['nameRequest']['nrNumber']
```

Comment:
The namex object name this prop "nrNum".

Reply:
Should we change our code so it's the same?

Reply:
Right now, we pass a only three properties from the name request ( nrNumber, legalName and legalType). If we make a schema change, we have to make the corresponding changes in all other components as well like filer, reports etc,

Reply:
Thor said we're going with "nrNumber" everywhere.